packages/manager/modules/nutanix/src/dashboard/nodes/node/module.js

```diff
@@ -0,0 +1,23 @@
+import angular from 'angular';
+import '@ovh-ux/manager-core';
+import '@uirouter/angularjs';
+import 'angular-translate';
+
+import component from './component';
+import routing from './routing';
+import ipmi from './ipmi';
+
+const moduleName = 'ovhManagerNutanixNode';
```

Comment:
If we pay attention, this module is using:

- `breadcrumb` from its routing section
- `<oui-` component from its template
- `alerter` directive from its template

All come from multiple modules that need to be added here if we want to keep standalone modules.

Reply:
Sorry I did not get it properly. All modules are like this right? whats different here and what the change required?

Reply:
From `ovhManagerNutanixNode` module point of view we are using multiple directives, components or resolvable.

All those items come from other modules that need to be imported and added as module dependency.

Here we can noticed that `ovhManagerNutanixNode` module has a dependency to following modules:
- `ngUiRouterBreadcrumb`
- `oui`
- `ngOvhUtils`

Reply:
Thanks. Got it. I have added all the missing dependencies.